src/js/game/LevelMVC/LevelView.js

```diff
@@ -1179,6 +1185,16 @@ module.exports = class LevelView {
   }
 
   refreshActionPlane(positions) {
+    // We need to add indices to refresh if there are other blocks in the action plane that might
+    // conflict with the drawing of refreshed blocks.
+    for (let i = 0; i < positions.length; ++i) {
+      if (positions[i][1] + 1 < this.controller.levelModel.actionPlane.height &&
+      this.controller.levelModel.actionPlane[this.controller.levelModel.actionPlane.coordinatesToIndex([positions[i][0], positions[i][1] + 1])].blockType !== "") {
```

Comment:
nit: could you break these out into clearly-named variables for clarity?

Reply:
What do you mean here? positions is an array of [x,y] elements, and that loop is finding if any element that exists in position has another element below it (because refreshing something that has an overlap from a southern index causes a draw conflict for like 1 frame) and then adds that southern element to the list-to-refresh (if you refresh both, the draw conflict doesn't manifest).

I can take an extra step to clarify it a bit, but do let me know if you have something specific in mind 👌

Reply:
Something along the lines of

```javascript
const positionBelow = [positions[i][0], positions[i][1] + 1];
const isPositionBelowValid = positionBelow[1] < this.controller.levelModel.actionPlane.height;
if (!isPositionBelowValid) {
  continue;
}

const indexBelow = this.controller.levelModel.actionPlane.coordinatesToIndex(positionBelow)
const isBlockBelow = this.controller.levelModel.actionPlane[indexBelow].blockType !== ""
if (isBlockBelow) {
  positions.push(positionBelow);
}
```

would be grand

Reply:
Ahhh alright, I can break this up even more. I stopped a few short of that, just explicitly defining that we were looking at blocks.

Reply:
For a codebase like this one that gets looked at once a year by a team that might not be the same as the team that looked at it last year, there's no such thing as making your code _too_ explicit

Reply:
Roger that. I broke up all the pieces that looked rationally break-up-able? It reads pretty clear to me, but again I am more than happy to change anything else.

Reply:
looks great!